### DISCUSSION TITLE
Add screen scaling appearance modification options

### DISCUSSION THREAD

**Discussion by StalkyBoi43815** [ppy/osu#29885] [Ideas] [answered]
User should be able to modify the appearance of screen scaling with more freedom. For example, background dim should be able to be set as a value between 0% and 100%, like in gameplay. Background options should be added. For example, when to change background, how long to wait until doing so, if to change the background in the first place, and pick a background to be set, among a list of possible backgrounds which are currently randomly picked.

**Comment by peppy** [comment, marked as answer]
This level of customisation will likely never be provided. This is a game, not a linux window manager.

**Reply by StalkyBoi43815** [reply]
Cmon man don't ruin my dreams of a lazer DE😞 I mean I think it would make a very pretty DE or Plasma theme lol
